shark-cli/src/main/java/shark/Main.kt

```diff
@@ -44,12 +71,14 @@ fun printHelp() {
 
     analyze-process: Dumps the heap for the provided process name, pulls the hprof file and analyzes it.
       USAGE: analyze-process PROCESS_PACKAGE_NAME
+             (optional) -proguard_mapping PROGUARD_MAPPING_FILE_PATH
```

Comment:
I actually don't know, I just wanted it to look different than the "main options" because it's more like adding an additional parameter. But I started reading and it seems that it's pretty interesting topic!
This is an awesome response with links to good resources: https://softwareengineering.stackexchange.com/questions/307467/what-are-good-habits-for-designing-command-line-arguments

Maybe we should use something like: http://commons.apache.org/proper/commons-cli/ or http://jcommander.org/?

I think it should be done as a separate change, and for this one I'll change the name to match the dash  convention you started. Will that be ok?

Reply:
Yeah that seems cool too and it's in Kotlin :)